lib/vanagon/platform/windows.rb

```diff
@@ -1,6 +1,7 @@
 class Vanagon
   class Platform
     class Windows < Vanagon::Platform
+      attr_accessor :proj_root
```

Comment:
To be more specific about this, is it acceptable to add project as an argument to the call in: https://github.com/puppetlabs/vanagon/blob/021bfd97c1855da7566269839f43e7eb4b760168/lib/vanagon/project.rb#L305

And the platform specific files as listed: https://github.com/puppetlabs/vanagon/search?utf8=✓&q=generate_packaging_artifacts